tests/test_integrated_channels/test_blackboard/test_transmitters/test_learner_data.py

```diff
@@ -1,21 +1,24 @@
 # -*- coding: utf-8 -*-
 """
-Tests for Moodle learner data transmissions.
+Tests for Blackboard learner data transmissions.
 """
 import unittest
 
 import mock
 from pytest import mark
 
-from integrated_channels.blackboard.models import BlackboardLearnerDataTransmissionAudit
+from integrated_channels.blackboard.models import (
+    BlackboardLearnerAssessmentDataTransmissionAudit,
+    BlackboardLearnerDataTransmissionAudit,
+)
 from integrated_channels.blackboard.transmitters import learner_data
 from test_utils import factories
 
 
 @mark.django_db
 class TestBlackboardLearnerDataTransmitter(unittest.TestCase):
     """
-    Test MoodleLearnerDataTransmitter
+    Test BlackboardLearnerDataTransmitter
     """
```

Comment:
guess pylint won't be happy but this comment seems not needed, if pylint does not block shall we remove? otherwise, okay :)

Reply:
we get a `Missing module docstring (missing-module-docstring)` if it's removed, feel like we might as well have it instead of ignoring the rule in all the test files.